docs/source/guides/writer/chapters/writing.rst

```diff
@@ -178,7 +178,7 @@ sleep length to be used by some other script or data analysis tool::
 The whiteboard can and should be exposed by files generated by the available test result
 plugins. The ``results.json`` file already includes the whiteboard for each test.
 Additionally, we'll save a raw copy of the whiteboard contents on a file named
-``whiteboard``, in the same level as the ``results.json`` file, for your convenience
+``whiteboard``, in the ``test-results`` directory, for your convenience
```

Comment:
Nice catch.  But I would change this slightly to make it extra clear that it's under one specific test's result directory.  So, in the end it is: ``test-results/$TEST_ID/whiteboard`.

Reply:
Yes, It will be much clearer.